runtime/spark/scripts/benchmark.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+sudo apt-get install -y git
+export USER_HOME=/home/$(whoami)
+SOFTWARE_HOME=$USER_HOME/runtime/benchmark-tools
+sudo mkdir -p $SOFTWARE_HOME
+sudo chown $(whoami):$(whoami) $SOFTWARE_HOME
+
+function install_sbt() {
+  sudo apt-get update
+  sudo apt-get install apt-transport-https curl gnupg -yqq
+  echo "deb https://repo.scala-sbt.org/scalasbt/debian all main" | sudo tee /etc/apt/sources.list.d/sbt.list
+  echo "deb https://repo.scala-sbt.org/scalasbt/debian /" | sudo tee /etc/apt/sources.list.d/sbt_old.list
+  curl -sL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | sudo -H gpg --no-default-keyring --keyring gnupg-ring:/etc/apt/trusted.gpg.d/scalasbt-release.gpg --import
+  sudo chmod 644 /etc/apt/trusted.gpg.d/scalasbt-release.gpg
+  sudo apt-get update
+  sudo apt-get install sbt -y
+}
+
+function install_maven() {
+  sudo apt install maven -y
+}
+
+function install_spark_sql_perf() {
+  install_sbt
+  cd ${SOFTWARE_HOME}
+  if [ ! -d "spark-sql-perf" ]; then
+    git clone https://github.com/databricks/spark-sql-perf.git && cd spark-sql-perf
+  else
+    cd spark-sql-perf && git pull
+  fi
+  sbt package
+}
+
+function install_tpcds_kit() {
+  sudo apt-get install -y gcc make flex bison byacc git
+  cd ${SOFTWARE_HOME} && git clone https://github.com/databricks/tpcds-kit.git
+  cd tpcds-kit/tools
+  make OS=LINUX
+}
+
+function install_tpch_dbgen() {
+  sudo apt-get install -y make patch unzip
+  cd ${SOFTWARE_HOME} && git clone https://github.com/databricks/tpch-dbgen
+  cd tpch-dbgen && make clean && make;
```

Comment:
According to previous experience on Beaver, current TPC-H commit may fail to work. 
It would be better to revert to `046930914` of TPC-H